contracts/settings/AlloSettings.sol

```diff
@@ -0,0 +1,51 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+pragma solidity 0.8.17;
+
+import "@openzeppelin/contracts-upgradeable/access/OwnableUpgradeable.sol";
+
+contract AlloSettings is OwnableUpgradeable {
+
+  string public constant VERSION = "1.0.0";
+
+  uint8 public constant PERCENTAGE_PRECISION = 100;
```

Comment:
what if we use `1000` here? so that it's easier to think about `100_000` as `100%` and `1_000` as `1%`?

Reply:
yup smart